ntoskrnl/cache/copysup.c

```diff
@@ -121,7 +121,9 @@ CcFastCopyRead(IN PFILE_OBJECT FileObject,
                OUT PVOID Buffer,
                OUT PIO_STATUS_BLOCK IoStatus)
 {
-    UNIMPLEMENTED_DBGBREAK();
+	LARGE_INTEGER Offset;
+	Offset.QuadPart = FileOffset;
+	CcCopyRead(FileObject, &Offset, Length, FALSE, Buffer, IoStatus);
```

Comment:
This should use TRUE for the Wait parameter. Check out the already existing version: https://github.com/reactos/reactos/blob/master/ntoskrnl/cc/copy.c#L464-L490